lib/net/nforceif/src/sys_arch.c

```diff
@@ -91,7 +91,7 @@ sys_now(void)
   asm __volatile__ ("rdtsc; divl %1"
                     : "=a" (now)
                     : "r" (freq)
-                    : "%eax", "%edx");
+                    : "%edx");
```

Comment:
This change actually causes problems:

clang won't let me use eax as dirty register so I removed it, thinking that it would not touch it until it's used as output.
I've now checked the behaviour and eax will be used for the `freq` input. So the `freq` will be broken and the timer output will be very bizarre values.

A fix for this is to use `"b" (freq)` to force the `freq` into a non-eax/edx register.
Another alternative approach is to ditch this inline assembly altogether and use the `KeTickCount` kernel export which is probably exactly what this function asks for: ms since reboot.